app/views/items/show.html.erb

```diff
@@ -27,9 +27,9 @@
          </div>
          <div>
             <% if @seller.name && @seller.name!="" %>
-               <p> <%= link_to(@seller.name, user_path(:userID => @sellerID)) %> </p>
+               <p> <%= link_to(@seller.name, user_path(@sellerID)) %> </p>
             <% else %>
-               <p> <%= link_to(@seller.email, user_path(:userID => @sellerID)) %> </p>
```

Comment:
:userID was breaking :id in users controller show method

Reply:
does changing this break any tests?

Reply:
It broke one or two but i fixed them-- functionality is preserved